chainerui/_version.py

```diff
@@ -1 +1,3 @@
 __version__ = '0.2.0'
+
+_db_revision = 'c78d4004717f'
```

Comment:
🤔 

I want to avoid duplicate definition.

Reply:
We can get the head revision:

```
config = Config()
script = ScriptDirectory.from_config(config)
print(script.get_current_head()) # => c78d4004717f
```

cf. http://alembic.zzzcomputing.com/en/latest/api/script.html

Reply:
I fixed.

- sort out initializing alembic config
- remove revision hardcoded